### PR TITLE
updated default config

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
- fix https://github.com/RasaHQ/rasa/issues/6810
- use a NLU pipeline with only countvectors as features by default
- remove deprecated `MappingPolicy` from docs example
- fix https://github.com/RasaHQ/rasa/issues/6736
- add `FallbackClassifer` to default config (it uses the default values of the old `FallbackClassifier`)

**Status (please check what you already did)**:
- [x] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by wochinge** [comment]
@akelad We still got [this section](https://github.com/RasaHQ/rasa/blob/34ee64572af4ba89e764cde46911242292addab8/docs/docs/tuning-your-model.mdx#L31) in the docs. I'm unsure if I should remove it as it still makes sense. Wdyt?

**Review by alwx — APPROVED** [review]

**Comment by erohmensing** [comment]
i'm updating the docs page, give me 5 mins :)

File: docs/docs/tuning-your-model.mdx [hunk 1]
```diff
@@ -28,38 +28,36 @@ pipeline:
...
 For example, if you have a sentence like “I want to buy apples” in your training data, and Rasa is asked to predict
 the intent for “get pears”, your model already knows that the words “apples” and “pears” are very similar.
 This is especially useful if you don't have enough training data.
-The advantage of the [ConveRTFeaturizer](./components.mdx#convertfeaturizer) is that it doesn't treat each word of the user message independently, but
-creates a contextual vector representation for the complete sentence.
-However, `ConveRT` is only available in English.
 
-If your training data is not in English, but you still want to use pre-trained word embeddings, we recommend using
-the following pipeline:
+If you are getting started with a one of [spaCy's supported lanaguages](https://spacy.io/usage/models#languages),
```
Comment (erohmensing) [inline docs/docs/tuning-your-model.mdx:37]:
```suggestion
If you are getting started with a one of [spaCy's supported languages](https://spacy.io/usage/models#languages),
```

**Comment by erohmensing** [comment]
Would be good if someone could sanity check my recommendations but otherwise good to go 👍

**Comment by wochinge** [comment]
@erohmensing As Daksh fixed the URL https://github.com/RasaHQ/rasa/pull/6743/files I'd actually revert your changes, ok?

**Comment by erohmensing** [comment]
That URL does not work anymore AFAIK

**Pull request merged**